react/src/index.js

```diff
@@ -325,7 +321,7 @@ class App extends Component {
               />
               <Route
                 path="/products-join"
-                element={<ProductsJoin backend={BACKEND_URL} />}
+                element={<Products backend={BACKEND_URL} isJoin={true} />}
```

Comment:
I would suggest we undo this part (see other comment) and undelete ProductsJoin.js for the time being.

Reply:
Understood. I've reverted the changes with https://github.com/sentry-demos/empower/pull/374/commits/6d9b95ac3d881c9a4419ed6d8fcb17b1296827cf.